flink-runtime/src/main/java/org/apache/flink/runtime/state/IncrementalRemoteKeyedStateHandle.java

```diff
@@ -359,11 +357,12 @@ IncrementalRemoteKeyedStateHandle copy() {
                 stateHandleId);
     }
 
-    /** Create a unique key to register one of our shared state handles. */
+    /** Create a unique key based on physical id to register one of our shared state handles. */
     @VisibleForTesting
-    public SharedStateRegistryKey createSharedStateRegistryKeyFromFileName(StateHandleID shId) {
+    public SharedStateRegistryKey createSharedStateRegistryKey(StreamStateHandle handle) {
+        String keyString = handle.getStreamStateHandleID().getKeyString();
```

Comment:
WDYT about renaming the corresponding commit from 
`[FLINK-29913][checkpointing] make IncrementalRemoteKeyedStateHandle register shared state use PhysicalStateHandleID as register key`
to something like
`[FLINK-29913][checkpointing] Use PhysicalStateHandleID as a key for shared state of IncrementalRemoteKeyedStateHandle`
and describe the problem this change solves in the commit message?